### ISSUE TITLE
Translate Gccode interface in french / Traduire l'interface de Gccode en Français

### ISSUE THREAD

**Issue by MaximMartel** [wet-boew/cdts-sgdc#62]
Moved from GCCode: https://gccode.ssc-spc.gc.ca/iitb-dgiit/nw-ws/sgdc-cdts/issues/136

@StdGit created the issue

https://docs.gitlab.com/ee/development/i18n/index.html#translate-gitlab-to-your-language

À fin que les unilingues francophones puissent facilement utiliser Gccode (GitLab), il faudrait que celui-ci soit traduit.

Ce changement selon moi devrait être effectué avant que nous indiquions aux développeurs qu'ils peuvent ouvrir des problématiques dans gccode.

Note: Probablement que la traduction française existe, il faut trouver comment l'utiliser.

**Comment by MaximMartel** [comment]
@shawnthompson commented: 

This is where you should open a ticket. I searched and there wasn't any issue already opened, surprising.

**Comment by MaximMartel** [comment]
@s-laugh commented: 

You can change the language in your profile settings. https://docs.gitlab.com/ee/user/profile/preferences.html#language

If translations are missing or incorrect, you can submit changes for them https://translate.gitlab.com/

**Comment by MaximMartel** [comment]
@StdGit commented: 

Thx it helps a lot.

**Comment by MaximMartel** [comment]
@shawnthompson commented:

@StdGit if that answers your question please close this issue.

**Comment by nschonni** [comment]
@MaximMartel this was related to translating GitLab, and since this isn't hosted there anymore, it should probably get closed. GitHub UI translations is a whole other problem ...
